draft-ietf-radext-radiusdtls-bis.md

```diff
@@ -624,6 +624,30 @@ This practice causes increased complexity in the client application and increase
 RADIUS/DTLS clients SHOULD implement session resumption, preferably stateless session resumption as given in {{RFC5077}}.
 This practice lowers the time and effort required to start a DTLS session with a server and increases network responsiveness.
 
+# Implementation Guidelines
+
+The text above describes the protocol. In this section, we give additional implementation guidelines. These guidelines are not part of the protocol, but they may help implementors create simple, secure, and interoperable implementations.
```

Comment:
```suggestion
The text above describes the protocol. In this section, we give additional implementation guidelines. These guidelines are not part of the protocol, but they may help implementors create simple, secure, and efficient implementations.
```
I would argue that the "interoperable" part is the responsibility of the protocol specification. So I would not put "interoperable" here, but a different word. I'm not sure if "efficient" is the right one, maybe something along "predictable" in the meaning that administrators of different RADIUS/(D)TLS servers by different vendors can rely on a somewhat similar behavior under the same circumstances.